multiraft/multiraft_test.go

```diff
@@ -69,31 +73,21 @@ func newTestCluster(transport Transport, size int, t *testing.T) *testCluster {
 		}
 		state := newState(mr)
 		demux := newEventDemux(state.Events)
-		demux.start()
+		demux.start(stopper)
 		cluster.nodes = append(cluster.nodes, state)
 		cluster.tickers = append(cluster.tickers, ticker)
 		cluster.events = append(cluster.events, demux)
 		cluster.storages = append(cluster.storages, storage)
 	}
-	cluster.start()
+	cluster.start(stopper)
 	return cluster
 }
 
-func (c *testCluster) start() {
+func (c *testCluster) start(stopper *util.Stopper) {
 	// Let all the states listen before starting any.
 	for _, node := range c.nodes {
-		go node.start()
-	}
-}
-
-func (c *testCluster) stop() {
-	for _, node := range c.nodes {
-		node.Stop()
-	}
-	for _, demux := range c.events {
-		demux.stop()
+		go node.start(stopper)
```

Comment:
Doesn't each of these `go node.start` calls need to be paired with an AddWorker?